packages/core/src/card/Card.tsx

```diff
@@ -53,6 +58,7 @@ export const Card = forwardRef<HTMLDivElement, CardProps>(function Card(
       className={clsx(
         withBaseName(),
         withBaseName(variant),
+        accentPlacement && withBaseName(`accent${capitalize(accentPlacement)}`),
```

Comment:
We've not been using this syntax to achieve conditional classes. Could we change this to the object notation please.